test/routes/issueRoutesTest.js

```diff
@@ -134,22 +131,15 @@ describe('Issue requests', function(){
         });
         describe('Invaild params', function() {
 
-            let issue;
-            before((done) => {
-                issue = {
-                    title: 0,
-                    name: null
-                };
-                done();
-            });
+            const testIssueIdInvalid = 1;
     
             it('should not edit issue', function(done){
                 request('http://localhost:8000')
-                .put('/issues/5aba5d0301f3703b5c72045')
-                .send(issue)
+                .put('/issues/' + testIssueIdInvalid)
+                .send(testIssue)
                 .then(done, (err) => {
                     expect(err).to.not.be.null;
-                    expect(err).to.have.status(500);
+                    expect(err).to.have.status(500);                    
```

Comment:
If the context is invalid params then response shouldn't be 500 but 400. Take a look at [this](https://www.smartlabsoftware.com/ref/http-status-codes.htm)